### ISSUE TITLE
[Bug]: Issue templates are broken

### ISSUE THREAD

**Issue by grafst** [savannabits/filament-signature-pad#24]
### What happened?

I cannot leave a suggestion because the link leads to https://github.com/coolsam/signature-pad/discussions/new?category=ideas

### How to reproduce the bug

click on issues --> new issue --> request a feature

### Package Version

1.2.0

### PHP Version

82.20

### Laravel Version

10.32.1

### Which operating systems does with happen with?

macOS, Windows, Linux

**Comment by WildEgo** [comment]
Yeah, was gonna ask if it's possible to publish the view in order to modify it but it seems like discussions are disabled even tho they're in the templates
